File: 2/HyeonJeong/java_hw/src/hw_2021_01_06/HW04.java
package hw_2021_01_06;

public class HW04 {
    public static void main(String[] args) {
        // float 형 배열에 랜덤 값 3 개를 입력해보자!

        final int NUM = 3;

        float[] f_series = new float[NUM];

        for(int i = 0; i < f_series.length; i++){
            float randomNum = (float)(Math.random() * 10);
            f_series[i] = randomNum;

            System.out.printf("array[%d] = %.3f\n", i, f_series[i]);
        }

    }
}

File: JAVASCRIPT/HyeonJeong/JavaScript_Work/src/21_01_29/homework/HW04.js
import { findAllByTestId } from "@testing-library/react"

const HW04 = () => {
    const INIT = 1

    const START  = 0
    const END = 15

    const CRITERIA = 15

    const ERROR = -1

    var emp = []
    var empNum

    // 0 ~ 14 까지 15개를 채운다는 의미
    for(var i = START; i < END; i++) {
        empNum = Math.floor(Math.random() * CRITERIA) + INIT
        
        // indexOf()를 통해 특정 가ㅄ을 찾아줌
        // 만약 찾지 못한다면 에러의 의미로 -1을 리턴한다 
        // 실제 리턴하는 타입이 NUMBER에 해당하므로 --도 무방
        if(emp.indexOf(empNum) == ERROR) {
           emp.push(empNum)
           console.log("Homework04: %d - %d", i, emp[i++])
        }
    }

    return(
        <div className = "HW04">
            <p>
                Homework04
            </p>
        </div>
    )
}

export default HW04